raiden/network/proxies/token_network.py

```diff
@@ -1606,9 +1606,9 @@ def _close(
 
                     # These checks do not have problems with race conditions because
                     # `poll`ing waits for the transaction to be confirmed.
-                    mining_block = int(receipt_or_none["blockNumber"])
+                    mining_block = int(failed_receipt["blockNumber"])
```

Comment:
I think `blockNumber` is already an int

Reply:
perhaps. I just don't want to do any unnecessary changes. But I will try to confirm in a test.

Reply:
It#s used in other places at least. But we can also do this in a follow up: https://github.com/raiden-network/raiden/pull/4512/files#diff-d02574a7a7eca826412780e80a6326a3R2360

Reply:
Yeah checked and it is indeed integer and also only 2 places had the extra int() cast so removed them.